doc-sites/package-lock.json

```diff
@@ -5,29 +5,37 @@
   "requires": true,
```

Comment:
I'm js naive but shouldn't package.json be ignored?

Reply:
According to the [documentation](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json)
```
package-lock.json is automatically generated for any operations where npm modifies either the node_modules tree, or package.json. It describes the exact tree that was generated, such that subsequent installs are able to generate identical trees, regardless of intermediate dependency updates.
```

So we do need to commit both package.json and package.lock.json